update setVerbosity doc (#105)
DataDog/dd-sdk-android: docs/log_collection.md

    {{% /tab %}}
    {{< /tabs >}}
    

> When writing your application, you can enable development logs. All internal messages in the library with a priority equal or higher than the provided level will be logged to Android's Logcat.

Reviewer: ```suggestion
When writing your application, you can enable development logs. All internal messages in the library with a priority equal or higher than the provided level are then logged to Android's Logcat.
```

| `setLogcatLogsEnabled(true)`     | Set to `true` to use Logcat as logger.                                                                                                                                                                                              |
| `setDatadogLogsEnabled(true)`    | Set to `true` to send logs to Datadog.                                                                                                                                                                                              |
| `setLoggerName(<LOGGER_NAME>)`   | Set `<LOGGER_NAME>` as the value for the `logger.name` attribute attached to all logs sent to Datadog.                                                                                                                                   |

> | `setVerbosity(Log.INFO)`         | Set the verbosity of the library. All internal messages in the library with a priority equal or higher than the provided level will be logged to Android's Logcat.                                                                   |

Reviewer: ```suggestion
| `setVerbosity(Log.INFO)`         | Set the verbosity of the library. All internal messages in the library with a priority equal or higher than the provided level are logged to Android's Logcat.                                                                   |
```
